less/index.less

```diff
@@ -54,6 +54,113 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	max-width: 800px;
+	
 }
+p {
+	font-size: 13px;
+	line-height: 16px;
+}
+nav {
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	
+	height: 50px;
+	a {
+		margin-right: 15px;
+		color: #0f0f0f;
+		font-size: 12px;
+		
+
+	}
+	a:last-child {
+		margin-right: 45px;
+	}
+}
+Header {
+	height: 440px;
+	background-color: rgb(238, 238, 238);
+	margin: 0 42px 0 42px;
+	border-radius: 4px;
+}
+h1 {
+	font-size: 32px;
+	margin-top: 10px;
+	margin-bottom: 8px;
+	
+}
+
+img {
+	height: 180px;
+}
+
+section {
+	padding-left: 20px;
+	padding-bottom: 20px;
+	padding-top: 35px;
+}
+.bottom-section {
+	border-top: 1px solid black;
+	margin-left: 20px;
+	padding-left: 0;
+	margin-right: 20px;
+	padding-top: 20px;
+	padding-bottom: 50px;
+}
+button {
+	background-color: gray;
+	color: white;
+	width: 105px;
+	height: 30px;
+	margin-top: 18px;
+	font-size: 13px;
+	border-radius: 4px;
+	
+}
+.hope {
+	display: flex;
+	border: 1px red solid;
+	margin-left: 22px;
+	width: 100%;
+	margin: 0 auto;
+	
+	
+}
+.listen {
+	display: inline-block;
+	width: 48%;
+	border: 1px solid black;
+
+}
+h4 {
+	font-size: 24px;
+	margin-bottom: 7px;
+}
+.stuff {
+	display: inline-block;
+	border: 1px solid purple;
+	width: 48%;
+	padding-bottom: 18px;
+}
+h5 {
+	font-size: 18px;
+	color: black;
+	padding-bottom: 8px;
+}
+.stuff div {
+	border: 1px solid lightgrey;
+	padding: 8px 80px 8px 8px;
```

Comment:
instead of re-writing the selectors here you can utilize LESS's nesting features like you did with your nav styles